x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -266,11 +267,16 @@ static RoleDescriptor kibanaSystem(String name) {
                     .indices(".logs-endpoint.actions-*")
                     .privileges("auto_configure", "read", "write")
                     .build(),
-                // Osquery manager specific action responses. Kibana reads from these to display responses to the user.
+                // Legacy Osquery manager specific action responses. Kibana reads from these to display responses to the user.
                 RoleDescriptor.IndicesPrivileges.builder()
                     .indices(".logs-osquery_manager.action.responses-*")
                     .privileges("auto_configure", "create_index", "read", "index", "delete")
                     .build(),
+                // Osquery manager specific action responses. Kibana reads from these to display responses to the user.
```

Comment:
Question:

This comment mentions that Kibana needs to "read", but this is also granting several other permissions.

Are all of these other privileges needed? I see they were allowed on the previous index, but it's always good to ask!

Reply:
Thanks for your comment. You are right. After consideration, I believe these two will be sufficient. Appreciate it 👍

Reply:
Thank you for looking into this!